_posts/jwarrich/2016-02-03-Javairia-Chapter5-Exercises.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+author: jwarrich
+title: "Javairia's Chapter 5 Exercises"
+---
+
+Exercise 1:
+
+<iframe src="https://trinket.io/embed/python/4be273a700" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+Exercise 2:
+
+<iframe src="https://trinket.io/embed/python/957a09c6e6" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+Reflection: 
+This was an interesting exercise to do. In the second exercise where we had to list the max and min, I had to think about whether I wanted to create a function outside of the loop or do everything within the loop. Which choice would be the best? I ultimately chose to make functions for the min and max because I thought it would be easier to just call the functions within the loop that I had created for exercise 1. 
```

Comment:
Good thoughts regarding the functions, but you didn't **call** them.  So you program just returns the function object.  They also want to take a list, and you don't construct a list.  We'll go over this more in class.